docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,78 @@
+let signUpPassword = '';
```

Comment:
Can we add an inline comment explaining the need for a global mutable `string`?

Reply:
Done!